lib/bike_brigade/tasks/mailchimp_importer.ex

```diff
@@ -80,7 +81,7 @@ defmodule BikeBrigade.Tasks.MailchimpImporter do
         Riders.update_rider(rider, rider_attrs)
 
       nil ->
-        Riders.create_rider(rider_attrs)
+        Riders.create_rider_with_user(rider_attrs)
```

Comment:
Ah, so this is how riders are created - from mailchimp?

Reply:
Yup!